### PR TITLE
Recommend use of RC for use with docs

### PR DESCRIPTION
**Pull request by flaskr**

### CODE REVIEW

File: doc/build_apps/install_bin.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ These dependencies can be conveniently installed using the ``ansible`` playbooks
 Install
 -------
 
-CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_.
+CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_. It is highly recommended to run `wget` directly on a release candidate with version 2.x.x as the documents are already updated for that version.
```
Comment (eddyashton) [inline doc/build_apps/install_bin.rst:30]:
```suggestion
CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_. The documentation you are currently reading describes the `main` branch and behaviour available in the 2.x release candidates.
```

Comment (eddyashton) [inline doc/build_apps/install_bin.rst:30]:
```suggestion
CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_. The documentation you are currently reading describes the `main` branch and behaviour available in the 2.x release candidates.
```

**Review by eddyashton — APPROVED** [review]
I've reworded the comment slightly - the docs are versioned, so it's only the current/main docs which require a `2.x` build. Thanks for the contribution @flaskr!

**Comment by flaskr** [comment]
Sounds great!

**Pull request merged**